flashinfer/__init__.py

```
@@ -266,3 +267,47 @@
+
+
+# ---------------------------------------------------------------------------
+# Import-time version log: emit one line when FLASHINFER_LOGLEVEL >= 1 so
+# that crash logs contain the exact commit without any manual archaeology.
+# Respects FLASHINFER_LOGDEST (stdout / stderr / filepath) the same way
+# api_logging.py does; defaults to stdout.
+# ---------------------------------------------------------------------------
+def _log_import_version() -> None:
+    # Wrapped in a private function so no temp variables leak into the
+    # flashinfer module namespace.  Two-level protection:
+    #   inner try  – safely resolve the commit hash; falls back to "unknown"
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)